modules/core/src/passes/layers-pass.ts

```diff
@@ -24,6 +24,7 @@ export type LayersPassRenderOptions = {
   clearCanvas?: boolean;
   layerFilter?: (context: FilterContext) => boolean;
   moduleParameters?: any;
+  preRenderStats?: Record<string, any>;
```

Comment:
```suggestion
  /** statistics for extensions */
  preRenderStats?: Record<string, unknown>;
```

Reply:
These are not for extensions, but communication between effects.

Reply:
Well, that is exactly why this code needs a comment... I could never have guessed that from this name. 

If it is for communication, consider a slightly longer comments about the conventions?